client/types/disputes.d.ts

```diff
@@ -76,7 +76,7 @@ export type DisputeStatus =
 export interface Dispute {
 	status: DisputeStatus;
 	id: string;
-	evidence_details?: EvidenceDetails;
+	evidence_details: EvidenceDetails;
```

Comment:
This is a required field as per the [Stripe dispute API docs](https://stripe.com/docs/api/disputes/object#dispute_object-evidence_details).